pkgs/projectcalico/calicoctl/registry.yaml

```diff
@@ -0,0 +1,29 @@
+---
+packages:
+  - name: projectcalico/calicoctl
```

Comment:
```suggestion
  - name: projectcalico/calico/calicoctl
```

The package name should start with the repository name `projectcalico/calico` to update the package by Renovate without additional setting.

https://github.com/aquaproj/aqua-renovate-config